django/test/runner.py

```diff
@@ -514,6 +533,20 @@ def run(self, result):
     def __iter__(self):
         return iter(self.subsuites)
 
+    def initialize_suite(self):
+        if multiprocessing.get_start_method() == "spawn":
+            self.initial_settings = {
+                str(alias): connections[alias].settings_dict for alias in connections
```

Comment:
Maybe we just need to not call `str(alias)` here? If `alias` already works as a key in `connections`, why do we need to change it? That will avoid having to put `str()` calls everywhere.

Reply:
https://docs.djangoproject.com/en/4.0/ref/settings/#databases

I agree that we're expecting `alias` to be a str. 👍